### PR TITLE
fix(admin-ui): upload app ui

### PR DESCRIPTION
**Pull request by frdomovic**
# fix(admin-ui): upload app ui

### CODE REVIEW

File: node-ui/src/components/applications/ApplicationsTable.jsx [hunk 1]
```diff
@@ -92,7 +98,11 @@ export function ApplicationsTable({ applications, install, uninstall }) {
 
   return (
     <Table>
-      {applications && (
+      {applications.length === 0 ? (
```
Comment (MatejVukosav) [inline node-ui/src/components/applications/ApplicationsTable.jsx]: Are you sure that apps always exists?

File: node-ui/src/constants/en.global.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
             "headerDescText": "Description",
             "headerRepoText": "Repository",
             "uninstallButtonText": "Uninstall",
-            "installButtonText": "Install new application"
+            "installButtonText": "Install new application",
+            "noApplicationsText": "No applicatons found in the node."
```
Comment (MatejVukosav) [inline node-ui/src/constants/en.global.json]: Applications

Comment (MatejVukosav) [reply node-ui/src/constants/en.global.json]: No applications installed in the node.

File: node-ui/src/constants/en.global.json [hunk 3]
```diff
@@ -65,10 +67,14 @@
     },
     "uploadPage": {
         "title": "Upload Application",
-        "walletButtonText": "Add wallet account"
+        "walletButtonText": "Add wallet account",
```
Comment (MatejVukosav) [inline node-ui/src/constants/en.global.json:70]: This should be redefined imo. It is not clear why user needs to add new wallet.

File: node-ui/src/constants/en.global.json [hunk 4]
```diff
@@ -88,6 +94,7 @@
     },
     "addPackageForm": {
         "title": "Package Details",
+        "subtitle": "To successfully install the application, first you need to add the package details and procceed to add relase.",
```
Comment (MatejVukosav) [inline node-ui/src/constants/en.global.json:97]: We need better copy here @matejpribanic

File: node-ui/src/components/applications/ApplicationsTable.jsx (hunk 1 above)
Comment (frdomovic) [reply node-ui/src/components/applications/ApplicationsTable.jsx]: updated

File: node-ui/src/constants/en.global.json (hunk 3 above)
Comment (frdomovic) [reply node-ui/src/constants/en.global.json:70]: can i merge this and next comment and we fix it next week?

Comment (MatejVukosav) [reply node-ui/src/constants/en.global.json:70]: Sure, but should be properly redesigned with @matejpribanic inputs. 

Can you open a task for this and lead it with @matejpribanic ?

File: node-ui/src/components/applications/ApplicationsTable.jsx [hunk 5]
```diff
@@ -92,7 +98,11 @@ export function ApplicationsTable({ applications, install, uninstall }) {
 
   return (
     <Table>
-      {applications && (
+      {applications && applications.length === 0 ? (
```
Comment (MatejVukosav) [inline node-ui/src/components/applications/ApplicationsTable.jsx]: Shouldn't this rather be `if(!applications || applications.length===0)`  so you don't need to check later if applications exist

File: node-ui/src/constants/en.global.json (hunk 3 above)
Comment (frdomovic) [reply node-ui/src/constants/en.global.json:70]: https://calimero.atlassian.net/jira/software/projects/C20/boards/28?selectedIssue=C20-53

**Pull request merged**